Makefile

```diff
@@ -125,7 +125,7 @@ devenv: devenv-down
 
 	@cd devenv; \
 	./create_docker_compose.sh $(targets) || \
-	(rm -rf docker-compose.yaml; exit 1)
+	(rm -rf {docker-compose.yaml,conf.tmp,.env}; exit 1)
```

Comment:
Is `conf.tmp` related with this issue? If not, I would proposed to be removed from this pull request and create a separate pull request for this one.

Reply:
Yeah, it is related, some of the blocks do use it and `create_docker_compose.sh` utilize it as well. 

I didn't know about it before tho